cmd/stat.go

```diff
@@ -209,11 +210,22 @@ func createStatOutputFile(input string) *os.File {
 // createOutputFileWithSuffix creates an output file named after prefixPath
 // appended with suffix.
 func createOutputFileWithSuffix(prefixPath, suffix string) *os.File {
-	output, err := os.Create(prefixPath + suffix)
+	fname := prefixPath + suffix
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		die("failed to get hostname: %s", err)
+	}
+
+	output, err := os.Create(fmt.Sprintf("%s%s.%s.%d", prefixPath, suffix, hostname, os.Getpid()))
 	if err != nil {
 		die("failed to create output file: %s", err)
 	}
 
+	if err = os.Symlink(fname, output.Name()); err != nil {
```

Comment:
I think the args to Symlink are the wrong way round? Please actually test this PR works on a local run before I accept and put put in to production.

Reply:
You're correct; the order was backwards.